### PR TITLE
Feature/coersion fixes

### PR DESCRIPTION
**Pull request by connor4312**

### CODE REVIEW

File: lib/rules/number.js [hunk 1]
```diff
@@ -2,6 +2,15 @@ import SyncRule from '../types/syncRule';
 
 class NumberValidator extends SyncRule {
 
+    coerce(value) {
+        if (typeof value !== 'string') {
+            return undefined;
+        }
+
+        const parsed = Number(String(value));
```
Comment (SimonSchick) [inline lib/rules/number.js]: `String` call needed here?

**Comment by SimonSchick** [comment]
lgtm, lets get this in and 🔥 some backend code.

File: lib/rules/number.js (hunk 1 above)
Comment (JamyDev) [inline lib/rules/number.js]: I wouldn't think so since there's already a check higher up that would ensure it being a string.

**Comment by JamyDev** [comment]
lgtm pending that one comment from Simon

**Comment by SimonSchick** [comment]
lets get rollin

**Pull request merged**